src/main/java/org/helpberkeley/memberdata/Main.java

```diff
@@ -649,12 +649,13 @@ private static void doDriverMessages(
         request.postStatus(WorkRequestHandler.RequestStatus.Processing, "");
 
         try {
-            String version = ControlBlock.create(routedDeliveries).getVersion();
-            if (version.equals(Constants.CONTROL_BLOCK_VERSION_300)) {
+            ControlBlock cb = ControlBlock.create(routedDeliveries);
+            String version = cb.getVersion();
```

Comment:
Personal style observation. This variable is only being used in the formatting the exception arguments. To my eye, the code would be more readable without the variable and instead directly calling cb.getVersion() in format calls(), because we could do that without making those lines too long.